dominion/dominion_ui/src/components/Header/Navigation.js

```diff
@@ -0,0 +1,226 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Link, MenuList, MenuItem, IconButton } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+import withWidth, { isWidthDown } from '@material-ui/core/withWidth';
+import logo from '../../assets/images/logos/dominion-logo.png';
+import logoWithCountrySpace from '../../assets/images/logos/dominion-logo-country.png';
+import Dropdown, { CountriesButton } from './PortalDropdown';
+
+import Search from '../Search';
+
+import menuIcon from '../../assets/images/icons/menu.svg';
+import backIcon from '../../assets/images/icons/back.svg';
+import searchIcon from '../../assets/images/icons/location.svg';
+
+import Modal from '../Modal';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1
+  },
+  topMenuNav: {
+    flexWrap: 'nowrap',
+    padding: 0,
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      position: 'absolute',
+      top: '6.25rem',
+      left: 0,
+      display: 'none'
+    }
+  },
+  topMenuIcon: {
+    color: 'white',
+    display: 'none',
+    [theme.breakpoints.down('sm')]: {
+      display: 'inline-block'
+    }
+  },
+  topMenuIconImg: {
+    marginRight: '-0.625rem'
+  },
+  menuList: {
+    width: '100%',
+    display: 'flex',
+    letterSpacing: '0.175rem',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      paddingTop: '0.625rem'
+    }
+  },
+  menuListItem: {
+    [theme.breakpoints.down('sm')]: {
+      paddingLeft: 0,
+      paddingRight: 0
+    }
+  },
+  img: {
+    height: '45px',
+    maxWidth: 'unset'
+  },
+  link: {
+    color: '#fff',
+    textDecoration: 'none',
+    fontFamily: theme.typography.fontFamily,
+    fontWeight: '600',
+    '&:hover': {
+      color: '#e7e452',
+      textDecoration: 'none'
+    }
+  },
+  logoCountryName: {
+    position: 'absolute',
+    color: 'white',
+    top: '1.75rem',
+    left: '3.625rem',
+    margin: 0,
+    fontFamily: theme.typography.fontFamily,
+    fontWeight: 500,
+    fontSize: 'x-small'
+  }
+});
+
+class Navigation extends Component {
+  renderMenuList() {
+    const { classes } = this.props;
+    return (
+      <MenuList className={classes.menuList}>
+        {['About', 'Showcase', 'Resources', 'Contact'].map(menuTitle => (
+          <MenuItem item className={classes.menuListItem}>
+            <Link
+              variant="body1"
+              className={classes.link}
+              to={`/${menuTitle.toLowerCase()}`}
```

Comment:
`Showcase` doesn't have it's own page (not yet anyway). It should: 
 1. links to an `id` on the same page for landing and country pages e.g. `#showcase`,
 2. link to landing page showcase, e.g. `/#showcase`, if you click it from any of the other portal pages, or
 3. link to country page showcase , e.g. `/kenya#showcase`, if you click it from any of the other country pages (if any).